preferences/oracle_linux/metadata/metadata.yaml

```diff
@@ -0,0 +1,9 @@
+---
+apiVersion: instancetype.kubevirt.io/v1beta1
+kind: VirtualMachinePreference
+metadata:
+  name: metadata
+  annotations:
+    tags: "hidden,kubevirt,oracle-linux"
+    iconClass: "icon-oracle"
+    openshift.io/display-name: "Oracle Linux"
```

Comment:
we really need to be more consistent with our use of the `instancetype.kubevirt.io/arch` label, it's used on arch specific preferences but should also be used in cases like this where we support multiple. Might be something for a follow up issue and PR tbh.

Reply:
I agree. I can already label this preference if we want. But I would be a bit inconsistent with other multi-arch preferences. Something like:

```yaml
instancetype.kubevirt.io/arch: "amd64/arm64"
```
would be ok?

Reply:
`instancetype.kubevirt.io/arch: "amd64,arm64"` would be better, something for a follow up tbh as we'd also need to handle this in some of the tests IIRC.

Reply:
Sounds good!